pkg/controllers/disruption/consolidation.go

```diff
@@ -45,7 +45,7 @@ import (
 const consolidationTTL = 15 * time.Second
 
 // MinInstanceTypesForSpotToSpotConsolidation is the minimum number of instanceTypes in a NodeClaim needed to trigger spot-to-spot single-node consolidation
-const MinInstanceTypesForSpotToSpotConsolidation = 15
+var MinInstanceTypesForSpotToSpotConsolidation = 15
```

Comment:
What's the need for us to change this from a constant?

Reply:
Reverted. This was dynamically changed previously to avoid churns but is no more relevant.